app/models/concerns/applicable_to_nations.rb

```diff
@@ -0,0 +1,30 @@
+module ApplicableToNations
+  extend ActiveSupport::Concern
+
+  included do
+    validate :applicable_to_at_least_one_nation
```

Comment:
Also worth validating that the alternative URLs are valid URLs to prevent broken links?